README.md

```diff
@@ -935,20 +938,22 @@ speech-to-intent into a already existing audio processing pipeline.
 `Rhino` is created by passing a context file to its static constructor `create`:
 
 ```dart
-import 'package:rhino/rhino_manager.dart';
-import 'package:rhino/rhino_error.dart';
+import 'package:rhino_flutter/rhino_manager.dart';
+import 'package:rhino_flutter/rhino_error.dart';
+
+const accessKey = "{ACCESS_KEY}"  // AccessKey obtained from Picovoice Console (https://picovoice.ai/console/)
 
 void createRhino() async {
     try{
-        _rhino = await Rhino.create('/path/to/context/file.rhn');
-    } on PvError catch (err) {
+        _rhino = await Rhino.create(accessKey, '/path/to/context/file.rhn');
+    } on RhinoException catch (err) {
         // handle rhino init error
     }
 }
 ```
 
 To deliver audio to the engine, you must send audio frames to its `process` function.
-Each call to `process` will return a Map object that will contain the following items:
+Each call to `process` will return a `RhinoInference` instance with following variables:
```

Comment:
You may want to mention in the explanation below that isUnderstood is null if not finalized and intent and slots are null if not finalized or understood. Just make it clear why and when those are null